src/groth16/aggregate/prove.rs

```diff
@@ -400,18 +400,18 @@ pub(super) fn polynomial_evaluation_product_form_from_transcript<F: Field>(
     res
 }
 
-/// Compute the coefficients of the polynomial $\prod_{j=0}^{l-1} (1 + x_{l-j}(rX)^{2j})$
-/// It does this in logarithmic time directly; here is an example with 2
-/// challenges:
-///
-///     We wish to compute $(1+x_1ra)(1+x_0(ra)^2) = 1 +  x_1ra + x_0(ra)^2 + x_0x_1(ra)^3$
-///     Algorithm: $c_{-1} = [1]$; $c_j = c_{i-1} \| (x_{l-j} * c_{i-1})$; $r = r*r$
-///     $c_0 = c_{-1} \| (x_1 * r * c_{-1}) = [1] \| [rx_1] = [1, rx_1]$, $r = r^2$
-///     $c_1 = c_0 \| (x_0 * r^2c_0) = [1, rx_1] \| [x_0r^2, x_0x_1r^3] = [1, x_1r, x_0r^2, x_0x_1r^3]$
-///     which is equivalent to $f(a) = 1 + x_1ra + x_0(ra)^2 + x_0x_1r^2a^3$
-///
-/// This method expects the coefficients in reverse order so transcript[i] =
-/// x_{l-j}.
+// Compute the coefficients of the polynomial $\prod_{j=0}^{l-1} (1 + x_{l-j}(rX)^{2j})$
+// It does this in logarithmic time directly; here is an example with 2
+// challenges:
+//
+//     We wish to compute $(1+x_1ra)(1+x_0(ra)^2) = 1 +  x_1ra + x_0(ra)^2 + x_0x_1(ra)^3$
+//     Algorithm: $c_{-1} = [1]$; $c_j = c_{i-1} \| (x_{l-j} * c_{i-1})$; $r = r*r$
+//     $c_0 = c_{-1} \| (x_1 * r * c_{-1}) = [1] \| [rx_1] = [1, rx_1]$, $r = r^2$
+//     $c_1 = c_0 \| (x_0 * r^2c_0) = [1, rx_1] \| [x_0r^2, x_0x_1r^3] = [1, x_1r, x_0r^2, x_0x_1r^3]$
+//     which is equivalent to $f(a) = 1 + x_1ra + x_0(ra)^2 + x_0x_1r^2a^3$
+//
+// This method expects the coefficients in reverse order so transcript[i] =
+// x_{l-j}.
```

Comment:
Why removing docs ? I thought these comments would be parsed as docs where there are 3 "/"

Reply:
The compiler tries to compile them apparently, and it causes build errors :-(

Reply:
It's the doc tests. Code blocks are expected to be Rust by default. You can either make it not a code block (by not indenting that part) or you could put it into a code fence and specify a type that is not rust, e.g. `text` or `latex`.